src/engine_render.c

```diff
@@ -5324,31 +5324,31 @@ static void display_fast_drawlist(struct Camera *cam)
         {
             switch (item.b->kind)
             {
-            case QK_JontySprite:
+            case QK_JontySprite: // Creatures and things
                 draw_fastview_mapwho(cam, item.jonSpr);
                 break;
-            case QK_ClippedLine:
+            case QK_SlabSelector: // Selection outline box for placing/digging slabs
                 draw_clipped_line(item.unk13->p.field_0,item.unk13->p.field_4,item.unk13->p.field_8,item.unk13->p.field_C,item.unk13->p.field_10);
                 break;
-            case QK_StatusSprites:
+            case QK_CreatureStatusSprite: // Status flower above creature heads
                 if (pixel_size == 1)
                     draw_status_sprites(item.unk14->x, item.unk14->y, item.unk14->thing, 48*256);
                 else
                     draw_status_sprites(item.unk14->x, item.unk14->y, item.unk14->thing, 16*256);
                 break;
-            case QK_TextureQuad:
+            case QK_TextureQuad: // Textured polygons
                 draw_texturedquad_block(item.txquad);
                 break;
-            case QK_IntegerValue:
+            case QK_FloatingGoldText: // Floating gold text when placing or selling a slab
                 draw_engine_number(item.number);
                 break;
-            case QK_RoomFlagPole:
+            case QK_RoomFlagBottomPole: // The bottom pole part, doesn't affect the status sitting on top of the pole
                 draw_engine_room_flagpole(item.roomFlg);
                 break;
-            case QK_JontyISOSprite:
+            case QK_JontyISOSprite: // Possibly unused
```

Comment:
the key was one of these right? so comment could be updated

Reply:
fixed. i had done the comment for display_drawlist but forgot about display_fast_drawlist